tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+DEFAULT_CEPH_DIR = "/etc/ceph/"
+DEFAULT_CONF_PATH = "/etc/ceph/ceph.conf"
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host(ceph): Installer host
+    """
+    timeout, interval = 20, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(host).ceph.orch.ls(service_type="mon", format="json")
+        configs = loads(out)
+        for config in configs:
+            running = config.get("status", {}).get("running")
+            size = config.get("status", {}).get("size")
+        if running == size:
```

Comment:
This if block is checking last `running` and `size` configs, Is thit intended ?

Reply:
Yes, code is checking if last running and size are equal or not.

Reply:
Used this logic instead.